include/rocksdb/advanced_cache.h

```diff
@@ -621,4 +638,8 @@ class CacheWrapper : public Cache {
 // reservations
 extern const Cache::CacheItemHelper kNoopCacheItemHelper;
 
+// Useful for cache entries that just need to be copied into a
+// secondary cache, such as compressed blocks
+extern const Cache::CacheItemHelper kSliceCacheItemHelper;
```

Comment:
I think this only makes sense in secondary_cache.h. It doesn't really persist realistic cache entries, so it only makes sense in the context of inserting directly into secondary cache, with a fake cache entry.